kernel/Cargo.toml

```diff
@@ -56,6 +56,11 @@ tokio = { version = "1", optional = true, features = ["rt-multi-thread"] }
 [features]
 arrow-conversion = ["arrow-schema"]
 arrow-expression = ["arrow-arith", "arrow-array", "arrow-ord", "arrow-schema"]
+cloud = [
+  "object_store/aws",
+  "object_store/azure",
+  "object_store/gcp",
+]
```

Comment:
we could add http store as well as a feature - this will support WebDAV based file servers. Not sure that people use this for these kind of scenarios, but it should add no additional dependencies?

Reply:
Yep, makes sense, done.